ppci/lang/c/context.py

```diff
@@ -43,6 +43,7 @@ def __init__(self, coptions, arch_info):
         double_size = self.arch_info.get_size(ir.f64)
         double_alignment = self.arch_info.get_alignment(ir.f64)
         self.type_size_map = {
+            BasicType.VOID: (1, 1),
```

Comment:
Strange, the void type should not require a size. What happens without this line?

Reply:
I know what the problem was, I installed from pip and the version there was crashing for some reason. When installing from master, there's no need for this.